.github/workflows/ci.yml

```diff
@@ -95,7 +92,6 @@ jobs:
     runs-on: ubuntu-latest
     needs:
       - changed
-    if: ${{ needs.changed.outputs.versions-json-changed == 'true' }}
     permissions:
```

Comment:
This comment is out of place, but you can only put them in places where there are changes. 

My query is about checking out the packages and config repos below. Is that necessary? I can't see where they're used.

Reply:
Also, below `steps.versions.outputs.packages-version` is reference. There is no `packages-version` output set in `versions`. Is this a concatenation of `steps.versions.outputs.packages` and `version`?

Reply:
They are being checked out to confirm that they exist at that particular revision. If they fail then the job fails, which means the prerelease deployment will not continue. 

Regarding your next comment, whoops. No, I just hadn't renamed the variable to `packages` or `config`.